optimade/validator/validator.py

```diff
@@ -542,19 +596,109 @@ def test_multi_entry_endpoint(self, request_str):
                 _type,
             )
             response_cls = ValidatorEntryResponseMany
-        deserialized = self.deserialize_response(response, response_cls)
+
         self.test_page_limit(response)
-        self.get_single_id_from_multi_endpoint(deserialized)
+
+        deserialized = self.deserialize_response(
+            response, response_cls, request=request_str, optional=True
+        )
+
+        if deserialized:
+            self.get_single_id_from_multi_endpoint(deserialized, request=request_str)
+            self.test_data_available_matches_data_returned(deserialized)
+
+    @test_case
+    def test_data_available_matches_data_returned(self, deserialized):
+        """In the case where no query is requested, `data_available`
+        must equal `data_returned` in the meta response.
+
+        """
+        if deserialized.meta.data_available != deserialized.meta.data_returned:
+            raise ResponseError(
+                "No query was performed yet data_returned != data_available."
```

Comment:
Where does the comma go in this sentence (before or after `yet`)? :sweat_smile: 
```suggestion
                "No query was performed yet data_returned != data_available."
```

Reply:
I'll switch to ", but" :grin: